doc/sphinx/installation.rst

```diff
@@ -184,16 +184,26 @@ using the following commands:
     sudo rm -r ~/anaconda[23]
     sudo rm -r /Library/Python
 
+Install the command line tools (gcc, git, make, etc.) with either:
+
+.. code-block:: bash
+
+    sudo xcode-select --install
+
+or:
+
+.. code-block:: bash
+
+    sudo xcodebuild -license accept
```

Comment:
`sudo xcodebuild -license accept` does not install anything. If Xcode has been downloaded from the Mac App Store but never started, it accepts its license. Please change to something like "If you have already installed Xcode from the Mac App Store, run `sudo xcodebuild -license accept`. Else install the command line tools using `sudo xcode-select --install`."

Reply:
In 15a0799 I've replaced everything with links to the official instructions for MacPorts and Homebrew.

Reply:
you commented on an outdated diff